parsely_raw_data/__init__.py

```diff
@@ -28,3 +29,39 @@
     'stream',
     'utils',
 ]
+
+BOOLEAN_FIELDS = [
+    "flags_is_amp",
+]
+
+
+def normalize_keys(r, schema):
+    """Conform events to public schema: correct keys and proper value types."""
+    schema = schema or schema.SCHEMA
```

Comment:
If you're going to overwrite `None` values here, you might want to make `schema` optional above. That way it's clear someone doesn't always have to specify it. Then you should put it in the docstring that Falsy values are going to use `schema.SCHEMA`. Otherwise, this might not be clear to a user unless they look at the code to see what's going on.